lib/utils/get-latest-tag.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const { exec } = require('child_process');
+const { promisify } = require('util');
+
+const execAsync = promisify(exec);
+
+/**
+ * Gets latest tag from provided package name.
+ *
+ * @param {string} name - Package name.
+ * @returns {Promise<StandardOutput>} - Return latest version, if latest tag exists.
+ * @throws {Error} - Output failed JSON parse.
+ */
+async function getLatestTag(name) {
```

Comment:
Is the `async` function here necessary, since await is not used here, and the execAsync returns the promise? Not sure if this is the case where its a promise wrapping around another promise!

Reply:
Also applies to the other util functions